data/campaigns/Eastern_Invasion/scenarios/17b_All-In.cfg

```diff
@@ -0,0 +1,1663 @@
+#textdomain wesnoth-ei
+#
+# NOTE: This level is intended as a challenging final scenario.
+#       Players without sufficient high-level troops to recall
+#       are advised to select "The Duel" instead.
+#
+#   I would prefer keeping this scenario as difficult as possible, so that players feel like they have a goal to reach for
+#
+[scenario]
+    id=17b_All-In
+    name= _ "All-In"
+    map_file=17b_All-In.map
+    turns=unlimited
+    victory_when_enemies_defeated=no
+    next_scenario=18_Epilogue
+    {DISABLE_GRAND_MARSHAL}
+
+    {DEFAULT_SCHEDULE_MORNING}
+
+    {INTRO_AND_SCENARIO_MUSIC weight_of_revenge.ogg cry_from_elensefar.ogg}
+    {EXTRA_SCENARIO_MUSIC battle-epic.ogg}
+    {EXTRA_SCENARIO_MUSIC heavens-remix.ogg}
+    {EXTRA_SCENARIO_MUSIC the_city_falls.ogg}
+
+    [story]
+        [part]
+            story= _ "The morning sun shone bright upon the valley of the Weldyn, glinting off swords and lances innumerable, arrayed in tight formation inside the city walls."
+            music,background=silence.ogg,story/S17b.jpg
+        [/part]
+        [part]
+            story= _ "From the ranks of Wesnoth’s finest arose a cry, soft at first but then mounting in volume; a cry of desperate hope, a cry of empty eyeless skulls, a cry of fiery purpose and icy will."
```

Comment:
Not sure what is meant by 'a cry of empty eyeless skulls'. Maybe trying to convey their exhaustion but sounds too much like undead to me.

Reply:
Agreed.  I've never been entirely happy with this line, and would welcome any suggested improvements.

Reply:
I feel like conveying their exhaustion in this way doesn't make any sense, as "exhausted people shouting" feels vaguely oxymoronic. I'd just remove that element altogether.

If you really want to keep it, I guess referencing something like the deadness of their eyes, or their tired posture, or something.